src/styles/index.scss

```diff
@@ -33,6 +33,16 @@
   }
 }
 
+// The following two entries are used to set the height of the content view (core-view), under the
+// toolbar, to the maximum. And the skeleton needs to maintain the height 100% otherwise its children
+// nodes with height: 100% or auto won't be displayed at all.
+#core-view {
+  height: calc(100vh - 48px);
+}
```

Comment:
I wonder if we should leave it here, or move it to another element. At the moment that removes `48px` from the area in `views/Dashboard.vue`, as we don't have a Toolbar visible. Though it doesn't seem to impact the user, that's still not good I think?

Reply:
(I would need a chat with you to understand the problem here, if it matters).